charts/kong/values.yaml

```diff
@@ -282,6 +283,7 @@ proxy:
     enabled: true
     servicePort: 443
     containerPort: 8443
+    appProtocol: true
```

Comment:
If it is used for enabling anything, the name should be more descriptive.

Reply:
So the `appProtocol` here is set to `true` to configure the `appProtocol` field to a certain value (same as actual protocol used), `false` to leave it empty? I think this is not direct at to configure it to be the actual value used, so I think here should use a more descriptive key, or add clear comment for it.

Reply:
@randmonkey hi
what do you think about key name `appProtocolEnable`  ?

Reply:
> @randmonkey hi what do you think about key name `appProtocolEnable` ?

IMO `setAppProtocol` would be clearer.

Reply:
> > @randmonkey hi what do you think about key name `appProtocolEnable` ?
> 
> IMO `setAppProtocol` would be clearer.

agree
added changes